composer.json

```diff
@@ -12,10 +12,8 @@
         }
     ],
     "require": {
-        "phpspec/prophecy": "~1.5"
-    },
-    "require-dev": {
-        "atoum/atoum": "~2.0"
+        "phpspec/prophecy": "~1.5",
+        "atoum/atoum": "^2.0"
```

Comment:
`atoum/atoum` is really a development dependency here. If you are using atoum and you would like this extension, atoum is already present. However, to test this extension alone, you need atoum, but as a development dependency.